ouroboros-consensus/src/Ouroboros/Consensus/Block/Forging.hs

```diff
@@ -159,11 +162,11 @@ data BlockForging m blk = BlockForging {
 -- It is possible to override this maximum transaction capacity with a lower
 -- value. We ignore higher values than the ledger state's max block size. Such
 -- blocks would be rejected by the ledger anyway.
-data MaxTxCapacityOverride
+data MaxTxCapacityOverride blk
```

Comment:
The comment on this type and on the `MaxTxCapacityOverride` constructor now seem to focus on bytes too much; could use some rewording now that it's more general.

Reply:
We neglected to catch this one during our call

Reply:
done